tests/prototest/use-reserved-id.proto

```diff
@@ -0,0 +1,78 @@
+// Sample .proto file that we can translate to the corresponding .fbs.
+
+option some_option = is_ignored;
+import "imported.proto";
+
+package proto.test;
+
+/// Enum doc comment.
+enum ProtoEnum {
+  option allow_alias = true;
+  NUL = 0;
+  FOO = 1;
+  /// Enum 2nd value doc comment misaligned.
+  BAR = 5;
+  // Aliases
+  FOO_A1 = 1;
+  BAR_A1 = 5;
+  FOO_A2 = 1;
+}
+
+/// 2nd table doc comment with
+/// many lines.
+message ProtoMessage {
+  reserved 200, 9 to 11, 1500;
+  // Ignored non-doc comment.
+  // A nested message declaration, will be moved to top level in .fbs
+  message OtherMessage {
+    optional double a = 26;
+    /// doc comment for b.
+    optional float b = 32 [default = 3.14149];
+
+    // Nested enum that aliases the outer one.
+    enum ProtoEnum {
+      NUL = 0;
+      FOO = 1;
+      BAR = 2;
+      BAZ = 3;
+    }
+
+    optional ProtoEnum foo_bar_baz = 33;
+  }
+  optional int32 c = 12 [default = 16];
+  optional int64 d = 1 [default = 0];
+  optional uint32 p = 40;
+  optional uint64 e = 2;
+  /// doc comment for f.
+  optional sint32 f = 3 [default = -1];
+  optional sint64 g = 4;
+  optional fixed32 h = 5;
+  optional fixed64 q = 6;
+  optional sfixed32 i = 7;
```

Comment:
I would make these protos that you added as small as possible, there is a lot of redundant things we are checking that add noise to the outputs.

We can either add new `messages` in the existing proto(s) that test certain properties you want to test, or if you want separate files (which I would vote no since we have so many test files already) make them as compact as possible.

Reply:
I should make separate small files, because they have serious problem such as using reserved-id, using id twice etc.
So they can not be parsed and fbs file wont be generated